test/e2e-test-application/src/app/project/drawer/drawer.component.ts

```diff
@@ -10,7 +10,11 @@ import { linkManager, uxManager } from '@luigi-project/client';
 export class DrawerComponent implements OnInit {
   public linkManager = linkManager;
   public uxManager = uxManager;
-  public constructor() {}
+  public constructor() { }
 
-  public ngOnInit() {}
+  public ngOnInit() { }
+
+  openConfirmationModal() {
+    uxManager().showConfirmationModal({ body: 'Just a confirmation modal' }).then(() => { console.log('test'); })
```

Comment:
Just a tiny suggestion, maybe we could have some more explicit console.log text instead of "test". E.g. "the confirmation modal is shown"